src/Microsoft.Azure.WebJobs.Host/Bindings/ConverterManager.cs

```diff
@@ -49,13 +127,38 @@ private static string DefaultByteArray2String(byte[] bytes)
             string key1 = GetKey<TSrc, TDest, TAttribute>();
             if (_funcsWithAttr.TryGetValue(key1, out obj))
             {
+                var builder = obj as Func<Type, Type, Func<object, object>>;
+                if (builder != null)
+                {
+                    Func<object, object> converter = builder(typeSrc, typeDest); // converter 
+                    FuncConverter<TSrc, TAttribute, TDest> func3 = (input, attribute, context) =>
+                    {
+                        var output = converter(input);
+                        return (TDest)output;
+                    };
+                    return func3;
```

Comment:
Can you give better names to some of these locals like func3, key1, etc. for better readability?